lib/jobs/Scheduler.js

```diff
@@ -122,7 +122,13 @@ class Scheduler {
       .try(() => {
         logger.debug(`Checking configured Job Types & Registering Job Definitions for : ${this.jobTypeList}`);
         if (this.jobTypeList.length > 0) {
-          _.each(this.jobTypeList, (jobType) => this.define(jobType, JobFabrik.getJob(jobType)));
+          _.each(this.jobTypeList, (jobType) => {
+            try {
+              this.define(jobType, JobFabrik.getJob(jobType));
+            } catch (err) {
+              logger.error(`error occurred while regisering job defintion : ${jobType}`, err);
```

Comment:
@kamath-prasad Thanks for noticing it. I missed it. 

@jagadish-kb Can you please create a PR for this?